cmd/kubeadm/app/phases/controlplane/manifests_test.go

```diff
@@ -28,8 +28,10 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/google/go-cmp/cmp"
```

Comment:
we already import reflect. maybe we can use it instead of the extra import of cmp

Reply:
reflect should be fine for slices, but i don't insist on not using cmp

Reply:
Gotcha, I will switch to reflect in the next iteration.

Reply:
@neolit123 have switched to reflect.